assets/src/pages/room/components/MediaControlButtons.tsx

```diff
@@ -100,14 +103,14 @@ const getAutomaticControls = (
           screenSharingStreaming.setActive(true);
         },
       },
-  //TODO enable when chat is implemented
-  // {
-  //   id: "chat",
-  //   icon: Chat,
-  //   hover: "Open the chat",
-  //   className: neutralButtonStyle,
-  //   onClick: () => undefined,
-  // },
+  {
+    id: "chat",
+    icon: Chat,
+    hover: isSidebarOpen ? "Close the chat" : "Open the chat",
```

Comment:
We should think of what to say here, but since we don't have chat this isn't a helpful tooltip. I'd stick to "Close/Open the sidebar" for now